src/main/java/com/jfrog/ide/idea/scan/SourceCodeScannerManager.java

```diff
@@ -131,13 +148,37 @@ public void onThrowable(@NotNull Throwable error) {
         executor.submit(createRunnable(sourceCodeScanTask, latch, log));
     }
 
-    private void sourceCodeScanAndUpdate(ProgressIndicator indicator, Runnable checkCanceled, Logger log) {
+    private void sourceCodeScanAndUpdate(ProgressIndicator indicator, Runnable checkCanceled, Logger log) throws IOException {
         indicator.setText("Running advanced source code scanning");
+        JFrogApplicationsConfig projectConfig = parseJFrogApplicationsConfig();
+
+        if (projectConfig != null) {
+            for (ModuleConfig moduleConfig : projectConfig.getModules()) {
```

Comment:
Can we run them in tasks? Running each scanner serially may take a lot of time

Reply:
We should do it in a separate PR in the near future.